### ISSUE TITLE
Freeze issue when idle

### ISSUE THREAD

**Issue by alexbohariuc** [cmer/gigabyte-z390-aorus-master-hackintosh#65]
Hey, @cmer i switched to OpenCore a while ago, got the same configuration as yours, but i get freezes while idling - after a minute or two. Screen saver kicks in and the i have to restart the pc to use it again. Some say it’s the ethernet card, but disabling it won’t solve the issue. 

Tried you EFI which to be honest is almost identical. I only have VGtab for my Vega 64. 

Did you encounter similar issues?
Thanks in advance!

**Comment by cmer** [comment]
I am definitely not experiencing this. Everything is flawless, including sleep. 

A couple of things to try:
- clear your nvram
- watch my bios setup video on YouTube and copy my settings? https://youtu.be/-cMrL1mXGZM
- make sure you have cfg lock disabled. https://youtu.be/W4JXVNJsK98

**Comment by alexbohariuc** [comment]
Sleep wasn’t working with Clover either. Will try those links, thanks! Altought i’m starting to think it’s something related to the video card or even the Broadcomm WIFI/BT card, which is actually identical to yours. 

Thanks again!

**Comment by alexbohariuc** [comment]
So, the sleep problem is related to the Broadcomm BT card, i have the exact model as yours - will try “hacking” it and plug it into an external one. 

Do you think that i should remap the USB ports, even if i’m using yours on the same mobo?

Still testing to see if the freeze problem is related to the BT card.

**Comment by alexbohariuc** [comment]
Nope, still freezes with nvram reset, cfg lock disabled and the same exact bios settings as yours.

**Comment by alexbohariuc** [comment]
I tried disabling the ethernet and removing intelmausiethernet (read that it might be the problem) but it still freezes. I’m kinda out of options, will check if it’s related to the video card, but i have no idea what else should i do.

@cmer besides taking your EFI, setting the BIOS and disabling cfg lock are there any other things i should care of? Should i generate my own SSDTs? 

Also, when booting intro OC bootloader i see this error:

OC: Setting NVRAM 4D1FDA02-38C7-4A6A-9CC6-4BCCA8B30102:rtc-blacklist - Not Found

Can it be related to my issue?

**Comment by alexbohariuc** [comment]
Ok, sorry for flooding, but apparently i found the solution:

Replaced IntelMausiEthernet with IntelMausi from acidanthera.
Fixed the VGtab Kext injection (wasn’t defined in Kernel -> Add) - using it for my Vega64.

It doesn’t freeze anymore. 

But, the usb from the BT Card prevents sleep (tried with the internal connector and also on one of the external ports). Any ideas?

**Comment by cmer** [comment]
Would you mind testing again with `IntelMausiEthernet` just to see if that's the actual cause? I don't have any freezes here. It'd help other people pinpoint the problem if they experience the same thing.

What's your BT card? Mine sleeps fine.

**Comment by cmer** [comment]
What BIOS version are you using?

**Comment by alexbohariuc** [comment]
I’m using the latest bios version - F11C.
However:

IntelMausiEthernet.kext it’s not causing the issue. It’s actually related to watchdog: 

`userspace watchdog timeout: no successful checkins from com.apple.WindowServer in 120 seconds `

Which it seems to be an issue that happens to some people on iMacs or even MBP.

The only difference is that when freezing IntelMausi doesn’t disconnect my internet connection. The weird thing is that when the system is freezing with IntelMausiEthernet my router disconnects from internet (PPPOE connection).

I still have to solve some things when i’ll have time to test: 

BT card is waking up the PC from sleep - i tried different usb ports. Even if the internal is set to 255 it wakes up the system from sleep. Disconnecting the usb works. 

I’ll try remapping the usb, and then use a different BT card, altought i have the same exact card as you and mobo. I’m a bit puzzled here.

The freezing issue: i stil don’t have a solution, i think it’s related to the video card, however i’m not willing to invest in a Radeon VII or something similar right now. I spent some money on my Vega64.

**Comment by nasedkinpv** [comment]
I don't know if it will help, but it makes sense to tell. I had friezes on the Aorus Z390 Pro, absolutely unbearable and annoying, also for a long time looked for the reason, and luckily I found it: when I hanged 4 devices on one usb 2.0 bus and among them there was an external audio card, I got friezes, as soon as I switched the audio card to 3.0 bus, everything worked like clockwork.
At first I also sinned to the graphics card and for a long time I fiddled with usbmap did not help anything. 
good luck!

**Comment by alexbohariuc** [comment]
Hmmm, i don’t have more than 1 devices on one bus, but i do have an external Focusrite audio interface connected to one USB 2.0 port. Will try, thanks!

**Comment by riley21** [comment]
Sounds like an issue I used to have regarding sleep.  Turned out to be the bluetooth card for me.  I originally had https://www.ebay.ca/itm/Dual-Band-BCM94360CS2-PCI-E-867Mbps-802-11AC-BT4-0-Wifi-PCI-Express-Adapter-Card/282961128646

but it kept waking from sleep despite bluetooth working.

I switched to 
https://www.amazon.ca/dp/B07NRYGZ7Q/ref=pe_3034960_233709270_TE_item
+ a 3 antenna BCM94360CS2 card.  (my original link from newegg doesn't work any longer but the item description was "BroadCom BCM94360CS2 BCM94360CS2AX Wireless WIFI Bluetooth Card Apple 653-0023 A")

Now everything bluetooth/sleep related works fine.
